styleguide.css

```diff
@@ -364,22 +419,135 @@ p a {
 
 p a:after {
     position: absolute;
-    top:1rem;
+    top: 1rem;
     bottom: 0.5rem;
     right: 0;
     left: 0;
     display: block;
     background-color: #bde513;
     overflow: hidden;
-    content: "";
+    content: '';
     transform: translateY(35%);
     transition: all .3s;
-    z-index:-1;
+    z-index: -1;
 }
 
 p a:hover:after {
-    top:0;
+    top: 0;
     bottom: 0.5rem;
     transform: translateY(0);
     transition: all .3s;
 }
+
+/* Menu */
+
+button.menu_button {
```

Comment:
Why a underscore here and other classes use a dash?

Reply:
ok, dashes then